lib/fluent/plugin/out_forward.rb

```diff
@@ -178,7 +178,7 @@ def start
       # But it should be overwritten by ack_response_timeout to rollback chunks after timeout
       if @ack_response_timeout && @delayed_commit_timeout != @ack_response_timeout
         log.info "delayed_commit_timeout is overwritten by ack_response_timeout"
```

Comment:
OT. Adding `why` message is better for users to understand why `delayed_commit_timeout` is overwritten in out_forward.